src/Microsoft.Health.CosmosDb/Features/Storage/RetryExceptionPolicyFactory.cs

```diff
@@ -16,6 +16,10 @@ public class RetryExceptionPolicyFactory
         private readonly TimeSpan _minWaitTime;
         private readonly TimeSpan _maxWaitTime;
 
+        public RetryExceptionPolicyFactory()
```

Comment:
Is this used?

Reply:
You mean the empty constructor right?

Reply:
Yes, the empty constructor. I didn't see anything that was using it, but I may have missed where it's used.